client/network/src/discovery.rs

```diff
@@ -59,8 +59,6 @@ use libp2p::kad::handler::KademliaHandlerProto;
 use libp2p::kad::QueryId;
 use libp2p::kad::record::{self, store::{MemoryStore, RecordStore}};
 #[cfg(not(target_os = "unknown"))]
-use libp2p::swarm::toggle::Toggle;
-#[cfg(not(target_os = "unknown"))]
```

Comment:
What this `cfg` removed by accident? We still don't want to try to import mdns when compiling for `wasm`, do we?

Reply:
The diff is misleading. The `cfg` above the two removed lines should now apply to `use libp2p::mdns::{Mdns, MdnsEvent};`.

Reply:
Oh, of course. My bad.